soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -705,7 +705,6 @@ void Randomizer::ParseRandomizerSettingsFile(const char* spoilerFileName) {
                     case RSK_SHUFFLE_COWS:
                     case RSK_SHUFFLE_ADULT_TRADE:
                     case RSK_SHUFFLE_MAGIC_BEANS:
-                    case RSK_RANDOM_MQ_DUNGEONS:
```

Comment:
this was never used as a part of the save context, and was also being parsed incorrectly as a boolean on/off value